toolkit/tools/pkg/rpmssnapshot/rpmssnapshot.go

```diff
@@ -171,7 +169,7 @@ func (s *SnapshotGenerator) readBuiltRPMs(specPaths []string, defines map[string
 		specDirPath := filepath.Dir(specPath)
 
 		go func(pathIter string) {
-			builtRPMs, err = rpm.QuerySPECForBuiltRPMs(pathIter, specDirPath, buildArch, defines)
+			builtRPMs, err := rpm.QuerySPECForBuiltRPMs(pathIter, specDirPath, buildArch, defines)
```

Comment:
I've started using something like `queryErr` instead of `err` to make the difference between the local and function level err more clear.

Reply:
Sounds good!